_unit-test/create-docker-volumes-test.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 source "$(dirname $0)/_test_setup.sh"
+set -ex
 
-expected=7
+expected=14
```

Comment:
test has been failing for some time now:

Our docker volumes should be
```
sentry-clickhouse
sentry-data
sentry-kafka
sentry-postgres
sentry-redis
sentry-self-hosted_sentry-clickhouse-log
sentry-self-hosted_sentry-kafka-log
sentry-self-hosted_sentry-nginx-cache
sentry-self-hosted_sentry-secrets
sentry-self-hosted_sentry-smtp
sentry-self-hosted_sentry-smtp-log
sentry-self-hosted_sentry-zookeeper-log
sentry-symbolicator
sentry-zookeeper
```

Reply:
Ok I take this back, expected number of volumes is 7. This test will fail locally though due to these containers that are brought up with docker compose, which isn't included in `create-docker-volumes.sh`
```
sentry-self-hosted_sentry-clickhouse-log
sentry-self-hosted_sentry-kafka-log
sentry-self-hosted_sentry-nginx-cache
sentry-self-hosted_sentry-secrets
sentry-self-hosted_sentry-smtp
sentry-self-hosted_sentry-smtp-log
sentry-self-hosted_sentry-zookeeper-log
```

Reply:
Test was failing due to `docker volume ls --quiet | grep '^sentry-.*' | wc -l` failing when there is are no docker volumes listed throwing an exception.

Reply:
I just changed the file to be more flexible so that it uses regex to determine if a list of expected volumes are in the actual docker volumes so that it passes both locally and in cli.